sdk/eventhub/azure-eventhub/azure/eventhub/_consumer_client.py

```diff
@@ -30,24 +30,21 @@
 
 
 class EventHubConsumerClient(ClientBase):
-    """The EventHubProducerClient class defines a high level interface for
+    """The EventHubConsumerClient class defines a high level interface for
     receiving events from the Azure Event Hubs service.
 
     The main goal of `EventHubConsumerClient` is to receive events from all partitions of an EventHub with
     load-balancing and checkpointing.
 
     When multiple `EventHubConsumerClient` operate within one or more processes or machines targeting the same
     checkpointing location, they will balance automatically.
-    To enable the load-balancing and / or checkpointing, checkpoint_store must be set when creating the
-    `EventHubConsumerClient`.
+    To enable the load-balancing, checkpoint_store must be set when creating the `EventHubConsumerClient`.
```

Comment:
why remove checkpointing?

Reply:
checkpoint can be done even if checkpoint_store is not set by user, we provide internal InMemoryCheckpointStore() for checkpoint now.

Reply:
updated for better explanation